apps/framework-cli/src/framework/python/wrappers/scripts/python_worker_wrapper/workflow.py

```
@@ -135,23 +135,21 @@ async def _monitor_workflow_history(
         log.info("Monitor task exiting because main task completed")
         return None
 
-    async def _execute_dmv2_activity_with_state(
-        self, dmv2wf: Workflow, task: Task, input_data: Optional[Dict] = None
+    async def _execute_activity_with_state(
+        self, wf: Workflow, task: Task, input_data: Optional[Dict] = None
     ) -> Union[List[Any], ContinueAsNewData]:
```

Ruff (0.14.11)

Use `X | Y` for type annotations

Convert to `X | Y`

(UP007)